src/components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,36 @@
+import { FormEvent } from 'react';
+import styles from './SearchBar.module.css';
+
+type SearchBarProps = {
+  placeholder: string;
+  searchLocation: string;
+};
+
+function SearchBar({
+  placeholder,
+  searchLocation,
+}: SearchBarProps): JSX.Element {
+  const handleSubmit = async (event: FormEvent) => {
+    event.preventDefault();
+    await search(searchLocation);
```

Comment:
I also cannot find `search`, what is it?

Reply:
Thats a function I will add later